dts/bindings/sensor/omron,2smpp03.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2020 Teslabs Engineering S.L.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Omron 2SMPP03 Pressure Gauge
+
+compatible: "omron,2smpp03"
+
+include: base.yaml
+
+properties:
+  io-channels:
+    type: phandle-array
+    required: true
+    description: IO channel associated with the amplified sensor voltage
+  ampl-gain:
```

Comment:
Can we spell this out to avoid ambiguities?  E.g. `amplifier-gain`?

Reply:
done